specification/dfp/resource-manager/Microsoft.DFP/preview/2021-02-01-preview/examples/ListOperations.json

```diff
@@ -0,0 +1,49 @@
+{
+  "parameters": {
+    "api-version": "2021-02-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "Microsoft.Dfp/instances/write",
+            "display": {
+              "provider": "Microsoft Dfp",
+              "resource": "Dynamic Fraud Protection",
+              "operation": "Create/Update Dfp instance",
+              "description": "Creates a DFP instance with the specified parameters or update the properties or tags."
+            }
+          },
+          {
+            "name": "Microsoft.Dfp/instances/delete",
+            "display": {
+              "provider": "Microsoft Dfp",
+              "resource": "Dynamic Fraud Protection",
+              "operation": "Delete Dfp instance",
+              "description": "Deletes an existing Dfp instance."
+            }
+          },
+          {
+            "name": "Microsoft.Dfp/checknameavailability/read",
```

Comment:
I believe this should be Microsoft.Dfp/locations/checkNameAvailability/action

Reply:
Fixed, thanks